dsgrid/config/dimensions.py

```diff
@@ -271,15 +271,14 @@ def check_file(cls, filename):
     @validator("file_hash")
     def compute_file_hash(cls, file_hash, values):
         if "filename" not in values:
-            # TODO #194
```

Comment:
Not reproducible. I think we were getting here because of other errors. This code is correct.